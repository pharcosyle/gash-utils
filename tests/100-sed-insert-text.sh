input='bar
baz
bam'

echo "$input" | \sed '/baz/ i Generated by Gash'
