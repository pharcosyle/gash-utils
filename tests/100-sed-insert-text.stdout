bar
Generated by Gash
baz
bam
